### DISCUSSION TITLE
Rotation around player with offset

### DISCUSSION THREAD

**Discussion by kennethrapp** [ramokz/phantom-camera#104] [General]
I want to rotate the camera based on the direction of the player or independently, preferably with an offset. I can't get anything to work - the closest I've gotten is putting a PhantomCamera inside a Node3D acting as a gimbal and setting the LookAt mode to mimic, but the camera only rotates in place, not around the player.

There's probably an obvious way to do this and I'm just not seeing it.

**Comment by ramokz** [comment]
Would you happen to have a video example, from another game, of what you're trying to achieve? Think I know what it is, but just want to make sure I understand it correctly.

**Comment by kennethrapp** [comment]
Any game where you can move the camera around the player?

Most third person controllers for Godot will let you do it. I found one example: [here](https://www.youtube.com/watch?v=UpF7wm0186Q) at about 5:40. You can see the position of the camera and model can be controlled independently of one another. This is usually done using a gimbal (example)[https://kidscancode.org/godot_recipes/3.x/3d/camera_gimbal/index.html] or spring arm.

**Comment by ramokz** [comment]
Attached a small sample scene, should be able to just add the extracted directory to your project with the plugin installed, is this what you were thinking of?

It's essentially just copied from the video you posted above, where you can move the character with `WASD`, the camera rotation around the character with the `mouse` and swap between two PhantomCameras with `spacebar`.

[Discussion_104.zip](https://github.com/ramokz/phantom-camera/files/11900404/Discussion_104.zip)

**Comment by kennethrapp** [comment]
That works fine. Don't know why I had such a problem with it. Thanks.
